### ISSUE TITLE
typo in dictionary06.txt

### ISSUE THREAD

**Issue by tagoh** [google/mozc#442]
There are the line in dictionary06.txt:
dictionary06.txt:こけらおとし   1847    1847    8952    柿落し

"柿"(U+67FF) isn't the "こけら". this is "かき". "こけら" is "杮" (U+676E)

**Comment by eEcaunt** [comment]
新明解国語辞典 第五版 has
```
     こけらおとし [4]【柿落とし・柿落し】
    〔新築の劇場や映画館の〕最初の興行。
```
and JMdict also has under 柿落し (こけらおとし) this:
```
     opening of a new theater (theatre)
```

**Comment by Arfrever** [comment]
- **杮** (U+676E): [https://en.wiktionary.org/wiki/杮](https://en.wiktionary.org/wiki/杮) （ハイ／**こけら**）
- **柿** (U+67FF): [https://en.wiktionary.org/wiki/柿](https://en.wiktionary.org/wiki/柿) （ジ／シ／**かき**）
(With larger size of font, difference between these characters is seen.)

**Comment by hiroyuki-komatsu** [comment]
Thank you for the report.
The latest commit no longer contain such entries of "杮 (U+676E) read as かき".

**Comment by Arfrever** [comment]
Fix was in commit a1dcadabd3a1c604e8beff1e45830c1d9adfce84.
